### PR TITLE
update(urls): RHICOMPL-1958 replace cloud with console in URLs

### PR DESCRIPTION
**Pull request by skateman**
Not entirely sure when this needs to be merged :thinking: 

## Secure Coding Practices Checklist GitHub Link
- https://github.com/RedHatInsights/secure-coding-checklist

## Secure Coding Checklist
- [x] Input Validation
- [x] Output Encoding
- [x] Authentication and Password Management
- [x] Session Management
- [x] Access Control
- [x] Cryptographic Practices
- [x] Error Handling and Logging
- [x] Data Protection
- [x] Communication Security
- [x] System Configuration
- [x] Database Security
- [x] File Management
- [x] Memory Management
- [x] General Coding Practices

### CODE REVIEW

File: config/settings/development.yml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 host_inventory_url: http://inventory-web:8081
-remediations_url: https://ci.cloud.redhat.com
-rbac_url: https://ci.cloud.redhat.com
+remediations_url: https://ci.console.redhat.com
+rbac_url: https://ci.console.redhat.com
```
Comment (vkrizan) [inline config/settings/development.yml:3]: I'm not sure if CI/QA will be available on the new location.

**Review by vkrizan — APPROVED** [review]
This is mostly a documentation fix (if we disregard CI/QA).

Approved.

File: config/settings/development.yml (hunk 1 above)
Comment (skateman) [reply config/settings/development.yml:3]: That's a bit scary :scream: :laughing:

**Review by akofink — APPROVED** [review]
I think we should keep CI+QA the same for now. Those environments will be EOL at the end of the month anyway.

**Pull request merged**